src/store/actions.ts

```diff
@@ -14,7 +14,7 @@ export default {
   setNextARDate ({ commit }, nextARDate: string) {
     commit('nextARDate', nextARDate)
   },
-  setNameRequest ({ commit }, nameRequest: object) {
+  setNameRequest ({ commit }, nameRequest: any) {
```

Comment:
It turns out objects have expected properties and methods. What we really want here is an "untyped object". `Any` seems to work fine and not cause type errors.